examples/inference-deployments/mpt/README.md

```diff
@@ -1,22 +1,57 @@
-## Model Description
+## Inference with MPT-7B 
 
-A state-of-the-art 6.7B parameter instruction finetuned language model trained by MosaicML. Check out the [MPT-Instruct huggingface repo](https://huggingface.co/spaces/mosaicml/mpt-7b-instruct) for more information.
+[MosaicML’s inference service](https://www.mosaicml.com/blog/inference-launch) allows users to deploy their ML models and run inference on them. In this folder, we provide an example of how to use MPT-7B, a state-of-the-art 6.7B parameter instruction fine-tuned language model trained by MosaicML for inference. Check out the [MPT-Instruct huggingface repo](https://huggingface.co/spaces/mosaicml/mpt-7b-instruct) for more information about the model.
 
-## Deploy
+You’ll find in this folder:
+
+- Model YAMLS - read [docs](https://docs.mosaicml.com/projects/mcli/en/latest/inference/inference_schema.html) for an explanation of each field.
+    - `mpt_7b.yaml` - a yaml to deploy [MPT-7B Base](https://huggingface.co/mosaicml/mpt-7b).
+    - `mpt_7b_instruct.yaml` - a yaml to deploy [MPT-7B Intstruct](https://huggingface.co/mosaicml/mpt-7b-instruct).
+    - `mpt_7b_storywriter.yaml` - a yaml to deploy [MPT-7B StoryWriter](https://huggingface.co/mosaicml/mpt-7b-storywriter).
+- Model handlers - these define how your model should be loaded and what should happen in a forward pass. You can use the default handlers here or write your custom model handler as per instructions [here](https://docs.mosaicml.com/projects/mcli/en/latest/inference/deployment_features.html#custom-model-handlers).
+    - `mpt_7b_handler.py` - a python script using DeepSpeed.
+    - `mpt_7b_ft_handler.py` - a python script using FasterTransformer.
+- `requirements.txt` - package requirements to be able to run these models.
+
+## Deploying your model
 
 To deploy, simply run `mcli deploy -f mpt_7b_instruct.yaml --cluster <your_cluster>`.
 
-## Input
+Once deployed, you can ping the deployment using
+```
+from mcli import ping
+ping('deployment-name')
+```
+to check if it is ready (status 200).
+
+More instructions can be found [here](https://docs.mosaicml.com/projects/mcli/en/latest/quick_start/quick_start_inference.html)
+
+
+
+## Sending requests to your deployment
```

Comment:
Can we also include example usage with langchain? https://github.com/hwchase17/langchain/blob/master/docs/modules/models/llms/integrations/mosaicml.ipynb

Would also be nice to make the different ways to query it expandable sections, like
"With MCLI"
"With curl"
"With langchain"
"With python"

Reply:
![Screenshot 2023-06-09 at 4 34 29 PM](https://github.com/mosaicml/examples/assets/1689437/77213409-5395-4f60-9f4c-dfcac9dc2a51)
Added!